elastic_app_search/client.py

```diff
@@ -289,6 +289,46 @@ def click(self, engine_name, options):
         endpoint = "engines/{}/click".format(engine_name)
```

Comment:
Please also sign the CLA

Reply:
I have applied your suggestions, but I wont be able to sign the CLA within the next few weeks.
I hope this PR can stay up, so I can merge it by then.